table_templates

```diff
@@ -513,6 +550,50 @@
    </table>
    <description>This table is how the ecu calculates 100% load at x rpm. The values shown are 100% load calculated from the base fuel schedule milliseconds. This is important for proper automatic transmission shifting. It needs to be adjusted so that the ecu sees at or near 100% when wide open throttle to correctly shift the automatic transmission.</description>
   </table>
+
+  
+  <table type="Switch" name="Knock sensor CEL (P0327, P0328)" category="OBD-II" sizey="4">
+   <description>When on, this enables the activation of the CEL when knock sensor is unplugged or shorted. </description>
+   <state name="on" data=" 00 A0 00 A0" />
+   <state name="off" data=" 00 20 00 20" />
+  </table>  
+
+<table type="Switch" name="Catalyst CEL Fix (P0420, P0430)" category="OBD-II" sizey="4">
+   <description>When enabled, this prevents the activation of the CEL when catalyst efficiency is below threshold. </description>
+   <state name="on" data=" 00 20 00 20" />
+   <state name="off" data=" 00 A0 00 A0" />
+  </table>
+  
+<table type="Switch" name="Catalyst DTC Max Threshold Fix (P0420, P0430)" category="OBD-II" sizey="2">
+   <description>When enabled, this changes the upper limit of the measured catalyst effeciency, preventing the DTC. </description>
+   <state name="on" data=" FF FF" />
+   <state name="off" data=" 60 60" />
+  </table>
   
+<table type="Switch" name="Catalyst DTC Min Threshold Fix (P0420, P0430)" category="OBD-II" sizey="8">
+   <description>When enabled, this changes the lower limit of the measured catalyst effeciency, preventing the DTC. </description>
+   <state name="on" data=" 01 01 01 01 01 01 01 01" />
+   <state name="off" data=" 32 32 32 32 32 32 32 32" />
+  </table>
 
+<table type="Switch" name="Secondary O2 Heaters DTC Fix (P0037, P0038, P0057, P0058)" category="OBD-II" sizey="2">
+   <description>When enabled, this changes the acceptable range of heater current and will not trigger a DTC for the secondary oxygen sensor heaters. </description>
+   <state name="on" data=" 00 FF" />
+   <state name="off" data=" 06 EC" />
+  </table>
+  
+<table type="Switch" name="VIAS Solenoid CEL (P1800)" category="OBD-II" sizey="2">
+   <description>When enabled, this enables the activation of the CEL when VIAS is unplugged. </description>
```

Comment:
heh, but why would you disable VIAS ?

Reply:
This is just for the CEL of the VIAS circuit. Factory state is disabled, just like the knock sensor and power steering switch circuits. This allows you to turn on the CEL for these DTCs, otherwise they're just "ghost" codes.